truss/cli/cli.py

```diff
@@ -728,8 +728,6 @@ def predict(
 @click.argument("script", required=True)
 @click.argument("target_directory", required=False, default=os.getcwd())
 def run_python(script, target_directory):
```

Comment:
this looks outdated due to the import. Is that right or am I missing something?

Reply:
oh -- I removed the try/except statement in https://github.com/basetenlabs/truss/pull/1059/files, we didn't need it anymore